taxi/forms.py

```diff
@@ -0,0 +1,47 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != 8:
+            raise forms.ValidationError(
+                "License number must be exactly 8 characters long.")
+
+        if (not license_number[:3].isupper()
+                or not license_number[:3].isalpha()):
+            raise forms.ValidationError(
+                "The first three characters must be uppercase letters.")
+
+        if not license_number[-5:].isdigit():
+            raise forms.ValidationError(
+                "The last five characters must be digits.")
+
+        return license_number
+
+
+class DriverCreateForm(DriverLicenseUpdateForm, UserCreationForm):
```

Comment:
The `DriverCreateForm` inherits from both `DriverLicenseUpdateForm` and `UserCreationForm`. This could lead to conflicts, especially if there are fields with the same name or conflicting validation logic. Please ensure that the inheritance order and field handling are correct.